openmc/deplete/independent_operator.py

```diff
@@ -330,14 +330,20 @@ def get_material_rates(self, mat_id, nuc_index, react_index):
             """
             self._results_cache.fill(0.0)
 
+            # Get volume in units of [b-cm]
+            volume_b_cm = 1e24 * self._op.number.get_mat_volume(mat_id)
+
             for i_nuc, i_react in product(nuc_index, react_index):
                 nuc = self.nuc_ind_map[i_nuc]
-                rxn = self.rxn_ind_map[i_react]
-
-                # Sigma^j_i * V = sigma^j_i * n_i * V = sigma^j_i * N_i
-                self._results_cache[i_nuc,i_react] = \
-                    self._op.cross_sections[rxn][nuc] * \
-                    self._op.number[mat_id, nuc]
+                rx = self.rx_ind_map[i_react]
+
+                # OK, this is kind of weird, but we multiply by volume in [b-cm]
+                # only because OpenMCOperator._calculate_reaction_rates has to
+                # divide it out later. It might make more sense to account for
+                # the source rate (flux) here rather than in the normalization
+                # helper.
```

Comment:
Yup, that's pretty much it. The `get_material_rates` method should be returning reaction rates, but in the `IndependentOperator` case we're just returning cross sections (because flux is accounted for later as part of the normalization).